src/script/lua_api/l_server.cpp

```diff
@@ -28,6 +28,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "server.h"
 #include "environment.h"
 #include "remoteplayer.h"
+#include "filesys.h"
```

Comment:
I feel bad pulling the whole of this in just for one function, this file is quite clean from depending on other things.
A "logic elsewhere" file…

Reply:
don't feel bad. if you need only one function described by filesys.h, only that will be compiled in (there is a big chance it already is compiled in by some other part of the project)

Reply:
I know :) Just was quite pleased at how "clean" this file was.